client/.prettierrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "plugins": ["prettier-plugin-tailwindcss"]
+}
```

Comment:
I thought this was a tailwind configuration 😅 

Could this file be moved to the root?